crypto3/.gitignore

```diff
@@ -250,7 +250,6 @@ TSWLatexianTemp*
 
 # Precompiled Headers
 *.gch
-*.pch
```

Comment:
Let's make single common .gitignore in the root of placeholder

Reply:
ok, I will do a general cmake-cleanup together with export statements unification